src/pageComponents/search/Results.jsx

```diff
@@ -69,8 +75,13 @@ class Results extends PureComponent {
     const { results } = this.props;
     return results.map((result, i) => {
       const { _index, _source: { title, url }, formattedDescription } = result;
+      const pathname = result._source.url;
       return (
-        <MediaWrapper key={ i } url={ url }>
+        <div key={ i }>
+        <Breadcrumbs path={ _index === 'guides' ?
+                            pathname : breadcrumbTexts[_index]}
```

Comment:
Like @systimotic stated it is not necessary to show breadcrumbs when 'Guides' or 'FreeCodeCamp' is searched. I'd replace the falsy condition with `'/'`. Otherwise we will need to update the Breadcrumbs.jsx file to handle null path prop.

Reply:
@Ethan-Arrowood Then, can we have breadcrumbs only if the result is from the guide?

Reply:
@dhanushuUzumaki it would be difficult and maybe confusing to show an external URL in a breadcrumb. So yes, we will only show breadcrumbs if the result is from the guide

Reply:
@Bouncey Okay. 😄 And shall I move the breadcrumb to the right side?

Reply:
I think that'll improve the UX @dhanushuUzumaki

Reply:
Thanks, @Ethan-Arrowood I will look into it and get back to you.

Reply:
@Ethan-Arrowood @Bouncey I have tried moving breadcrumbs to the right side of the title as well as below title. Which one should we go for?
Below Title:
![image](https://user-images.githubusercontent.com/14927280/31586765-0b1204e6-b1f3-11e7-83c5-9ec183aa5dfe.png)
Right of Title:
![image](https://user-images.githubusercontent.com/14927280/31586768-13511aac-b1f3-11e7-8305-c942ae340536.png)

And since the entire media is covered by an <a> tag and breadcrumb also have an <a> there is a dom validation error? any idea on how to resolve it might be helpful.

Thanks 😄

Reply:
Great work on this. I like the underneath more. Could you add some margin to each Search Result? I feel like the list seems squished together. A little bit of white space goes a long way in UX design.

Reply:
@Ethan-Arrowood Sure will do it. And do you have any idea for the dom validation error?

Reply:
<img width="563" alt="screen shot 2017-10-16 at 9 56 12 am" src="https://user-images.githubusercontent.com/16144158/31615671-4870107a-b258-11e7-9160-f451edfbb1dc.png">
Go to line 74 space 24 and remove the trailing white space. Re run the lint command and you should be good on that end.

Reply:
Hi, @Ethan-Arrowood. Sorry got some tied up at work and wasn't able to do it. I am not talking about the linting error. We have a rule as not to have an <a> tag inside an <a> tag. But since the media-wrapper has an <a> tag and we are adding breadcrumbs inside it which also has an <a> tag in browser console window we get a DOM validation error.. I was hoping if there is any way to solve it.

Reply:
Sorry @dhanushuUzumaki I am only just seeing this. Thank you for your patience.

There is no need for the search results breadcrumbs to have an `<a>` tag, as if should be linking to the same place as the search result itself. If `Breadcrumbs` can be refactored cleanly for this use case, that would be great.

Reply:
@Bouncey Can we have a prop to check whether to render it as a link or not?